lib/hammer_cli_foreman/compute_resource/vmware.rb

```diff
@@ -24,9 +25,11 @@ def compute_attributes
           ['annotation',           _('Annotation Notes')],
           ['scsi_controllers',     [_('List with SCSI controllers definitions'),
                                     '  type - ' + _('ID of the controller from VMware'),
-                                    '  key  - ' + _('Key of the controller (e.g. 1000)')
-                                    ].flatten(1).join("\n")],
-          ['boot_order',            _('Device names to specify the boot order')]
+                                    '  key  - ' + _('Key of the controller (e.g. 1000)')].flatten(1).join("\n")],
+          ['nvme_controllers', [_('List with NVME controllers definitions'),
+                                '  type - ' + _('ID of the controller from VMware'),
```

Comment:
It looks a bit strange that it's called a "type", but it requires an identifier. Is it an identifier of a controller type or a particular controller?